topics/cargo-generate/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: rust-app-template scaffolding rust-app-scaffolding
+created_by: Sven Assmann
+display_name: cargo-generate
+github_url: https://github.com/cargo-generate/cargo-generate
+related: rust, template, scaffold
+released: Oct, 2017
+short_description: Cargo, mage me a project.
```

Comment:
```suggestion
short_description: Cargo, make me a project.
```

Is this a typo?

Don't mind me if this is an intentional in-joke 😆

Reply:
good catch, was indeed not intended.